CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ option(FASTGLTF_ENABLE_DEPRECATED_EXT "Enables support for deprecated extensions
 option(FASTGLTF_DISABLE_CUSTOM_MEMORY_POOL "Disables the memory allocation algorithm based on polymorphic resources" OFF)
 option(FASTGLTF_USE_64BIT_FLOAT "Default to 64-bit double precision floats for everything" OFF)
 option(FASTGLTF_COMPILE_AS_CPP20 "Have the library compile as C++20" OFF)
+option(FASTGLTF_INSTALL "Install fastgltf" OFF) # For CI.
```

Comment:
Why add this option exactly? CMake install commands only configure a special installation target for the targets, so if you never run `cmake --install .` those commands won't ever run.

Reply:
My mistake. Reverted.